schema-engine/connectors/sql-schema-connector/src/introspection/introspection_helpers.rs

```diff
@@ -55,47 +55,8 @@ pub(crate) fn is_relay_table(table: TableWalker<'_>) -> bool {
             .any(|col| col.name().eq_ignore_ascii_case("stablemodelidentifier"))
 }
 
-/// If the table has `createdAt` and `updatedAt` fields.
-pub(crate) fn has_created_at_and_updated_at(table: TableWalker<'_>) -> bool {
-    let has_created_at = table.columns().any(|col| {
-        col.name().eq_ignore_ascii_case("createdat") && col.column_type().family == ColumnTypeFamily::DateTime
-    });
-
-    let has_updated_at = table.columns().any(|col| {
-        col.name().eq_ignore_ascii_case("updatedat") && col.column_type().family == ColumnTypeFamily::DateTime
-    });
-
-    has_created_at && has_updated_at
-}
-
-/// A special table to handle many to many relations in Prisma.
-pub(crate) fn is_prisma_join_table(t: TableWalker<'_>) -> bool {
-    is_prisma_1_point_0_join_table(t) || is_prisma_1_point_1_or_2_join_table(t)
-}
-
-/// A legacy Prisma 1 list table.
-pub(crate) fn is_prisma_1_or_11_list_table(table: TableWalker<'_>) -> bool {
-    table.columns().len() == 3
-        && table.columns().any(|col| col.name().eq_ignore_ascii_case("nodeid"))
-        && table.column("position").is_some()
-        && table.column("value").is_some()
-}
-
-/// A special legacy join table, seen in Prisma 1.1 and 1.2.
-pub(crate) fn is_prisma_1_point_1_or_2_join_table(table: TableWalker<'_>) -> bool {
-    table.columns().len() == 2 && table.indexes().len() >= 2 && common_prisma_m_to_n_relation_conditions(table)
-}
-
-/// A special legacy join table, seen in Prisma 1.0.
-pub(crate) fn is_prisma_1_point_0_join_table(table: TableWalker<'_>) -> bool {
-    table.columns().len() == 3
-        && table.indexes().len() >= 2
-        && table.columns().any(|c| c.name() == "id")
-        && common_prisma_m_to_n_relation_conditions(table)
-}
-
 /// If a relation defines a Prisma many to many relation.
-fn common_prisma_m_to_n_relation_conditions(table: TableWalker<'_>) -> bool {
+pub fn is_prisma_m_to_n_relation(table: TableWalker<'_>) -> bool {
```

Comment:
Can you make this `pub(crate)` instead? So it doesn't end up in the crate's public API.

Reply:
Yes! good point

Reply:
Done ✅